source/GameData.cpp

```diff
@@ -187,6 +187,8 @@ bool GameData::BeginLoad(const char * const *argv)
 		it.second.FinishLoading(true);
 	for(auto &it : persons)
 		it.second.FinishLoading();
+	for(auto &it : missions)
```

Comment:
Should be after `startConditions.FinishLoading()` and introduced by a different comment. The other calls are all finalizing ship definitions.

Reply:
I removed this because it is no longer needed after your other suggested change.